packages/vue/src/components/confirm-sign-up.vue

```diff
@@ -5,10 +5,10 @@
         <base-heading>
           {{ confirmSignUpHeading }}
         </base-heading>
-        <base-field-set :disabled="state.matches('confirmSignUp.pending')">
+        <base-field-set :disabled="actorState.matches('confirmSignUp.pending')">
           <user-name-alias
             :userNameAlias="true"
-            :userName="state?.context?.formValues[primaryAlias]"
+            :userName="actorState?.context?.formValues[primaryAlias]"
```

Comment:
The formValues no longer populated at this point. It needs to be retrieved from the `authAttributes`

```suggestion
            :userName="actorState?.context?.authAttributes?.username"
```

Reply:
Is it sometimes `user?.username` Might need to make that check not sure

Reply:
Ah, thanks for the catch! And you're right, it's `user.username` if coming directly from `signUp`. Making that adjustment along with yours.

Reply:
You're right, thanks!https://github.com/aws-amplify/amplify-ui/pull/180/commits/b422514114380de35fa7dfb1a86fa8343ba8053c